test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Admin/DatabaseAdminTests.cs

```diff
@@ -85,5 +85,25 @@ private LiveConnectionHelper.TestConnectionResult GetLiveAutoCompleteTestObjects
             requestContext.VerifyAll();
         }
 
+        /// <summmary>
+        /// Get database info test
+        /// </summary>
+        // [Fact]
```

Comment:
the other tests were failing in some of the CI runs and were commented out until the create db feature is done.  If this test is reliable you should leave it enabled.

---
In reply to: [120791129](https://github.com/Microsoft/sqltoolsservice/pull/370#discussion_r120791129) [](ancestors = 120791129)